### DISCUSSION TITLE
The android version has an unusable layout

### DISCUSSION THREAD

**Discussion by Owd31222** [godotengine/godot-proposals#9724] [Editor]
(English is not my first language so Sorry if it's hard to understand what I have written)

I use the android version because it's the only thing I have at my disposal since I don't have the money for a computer.
The game engine is made for PC and it's nearly impossible to use and understand on a phone screen. I tried multiple times to create something but I can't 

It's cluttered, 
every button is to small for a phone screen,
The panels on the side and on the bottom are always vizibile even when you don't need them(especially the lower one),
There's no space for the keyboard when you need to write something,
The auto fill for my phone is often overwrittes the auto fill of the script when I write code.

A few solution's I see could work are: 
First and in my opinion the most important one is allowing to change the size of the panels, (especially the lower one  because the only two options are you have that it's occupying half the screen or all of the screen),
Allowing to hide the panels you don't need them ,
Depending in which orientation is the screen reveales and hides different settings and panels.
Button resizing 

Please someone read this and listen to my rambling I really enjoy (trying) make games but the layout is so unfriendly I rarely finish a project.
![Screenshot_2024-05-11-00-01-10-346_org.godotengine.editor.v4.jpg](https://github.com/godotengine/godot-proposals/assets/96998775/470c54af-521c-4e77-bc85-b055803bd032)

![Screenshot_2024-05-10-23-31-44-093_org.godotengine.editor.v4.jpg](https://github.com/godotengine/godot-proposals/assets/96998775/e273fe52-7712-497a-aba1-8777d753b4ae)

![Screenshot_2024-05-10-23-20-04-334_org.godotengine.editor.v4.jpg](https://github.com/godotengine/godot-proposals/assets/96998775/b8310852-1e06-426d-a92a-b2915847639a)

**Comment by Calinou** [comment]
>  The panels on the side and on the bottom are always vizibile even when you don't need them(especially the lower one),

You can click the name of the currently active bottom panel to hide it. The active panel is the one with the name displayed in blue.

**Comment by Zireael07** [comment]
Regarding the keyboard issue I raised it as a separate ticket a long time ago, so long that I can't find the ticket number.

I can only second your other observations
